src/controllers/controlpickermenu.cpp

```diff
@@ -434,7 +434,28 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                tr("Add to Auto DJ Queue (replace)"),
                tr("Replace Auto DJ Queue with selected tracks"),
                libraryMenu, false, m_libraryStr);
-
+    addControl("[Library]",
+            "search_history_next",
+            tr("Select next search history"),
+            tr("Selects the next search history entry"),
+            libraryMenu,
+            false,
+            m_libraryStr);
+    addControl("[Library]",
+            "search_history_prev",
+            tr("Select previous search history"),
+            tr("Selects the previous search history entry"),
+            libraryMenu,
+            false,
+            m_libraryStr);
+    addControl("[Library]",
+            "search_history",
```

Comment:
Just `search_history` or `search_history_move`/`search_history_step`? I didn't check how this is handled by similar controls if there are any.

Reply:
I renamed it to search_history_move